### PR TITLE
Scheduler and Webserver with separate resource configuration

### PR DESCRIPTION
**Pull request by sriniarul**
# Summary
 Adding the resource configuration in values file let you deploy scheduler and webserver with different cpu and memory configurations

# Tests
 Deployed in local k8s cluster and verified that the custom resource configurations are working or not

### CODE REVIEW

File: charts/mageai/templates/webservice.yaml [hunk 1]
```diff
@@ -79,7 +79,7 @@ spec:
             timeoutSeconds: {{ .Values.readinessProbe.timeoutSeconds }}
           {{- end }}
           resources:
-            {{- toYaml .Values.resources | nindent 12 }}
+            {{- toYaml .Values.webServer.resources | nindent 12 }}
```
Comment (wangxiaoyou1993) [inline charts/mageai/templates/webservice.yaml:82]: can you make it backward compatible?

Comment (sriniarul) [reply charts/mageai/templates/webservice.yaml:82]: i cant understand why we required backward compatibility. While installation its gonna use the default resource configuration.

Comment (wangxiaoyou1993) [reply charts/mageai/templates/webservice.yaml:82]: Currently, user configures the web server and scheduler resources using the resources configuration in values.yaml
If they upgrade the helm chart version, their current configuration will not be applied anymore.

So in your change, you should fallback to the `.Values.resources` if `.Values.webServer.resources` is not present.

Comment (sriniarul) [reply charts/mageai/templates/webservice.yaml]: @wangxiaoyou1993 fallback support applied

**Comment by hanslemm** [comment]
@wangxiaoyou1993 can we release this?

**Review by wangxiaoyou1993 — APPROVED** [review]

**Pull request merged**